test/src/edit/RotateAndScaleHandleSpec.js

```diff
@@ -20,6 +20,10 @@ describe("L.RotateAndScaleHandle", function() {
 		});
 	});
 
+	it.skip("Should not distort the image during scaling", function () {
```

Comment:
We can add a test for this, I guess in a seperate issue?

Reply:
Thank you that's so nice!! yes but I'm not sure it should fully pass yet. because the image size passed in for `edgeMinWidth` is hardcoded, and the person can make edits to the image and change the shape and then scale. It would be a whole separate issue to work on for sure